src/Core/src/Platform/Tizen/TransformationExtensions.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using ElmSharp;
+using ERect = ElmSharp.Rect;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		static bool _movedCallbackEnabled = false;
+
+		public static void UpdateTransformation(this EvasObject nativeView, IView? view)
+		{
+			if (view == null)
+				return;
+
+			// prepare the EFL effect structure
+			ERect geometry = nativeView.Geometry;
+			EvasMap map = new EvasMap(4);
+			map.PopulatePoints(geometry, 0);
+
+			bool changed = false;
+			view.ApplyScale(map, geometry, ref changed);
+			view.ApplyRotation(nativeView, map, geometry, ref changed);
+			view.ApplyTranslation(map, geometry, ref changed);
+
+			nativeView.IsMapEnabled = changed;
+			if (changed)
+			{
+				nativeView.EvasMap = map;
+				if (!_movedCallbackEnabled)
+				{
+					_movedCallbackEnabled = true;
+					nativeView.Moved += OnMoved;
+				}
+			}
+			else
```

Comment:
`nativeView.Moved` 콜백에 대한 처리를 ViewHandler로 옮기는게 맞아 보입니다.

``` c#

interface IMoveController {
   void SetMovedDelegate(Action moved);
   void RemoveMovedDelegate();
}

public abstract class EViewHandler<TVirtualView, TNativeView> : ViewHandler<TVirtualView, TNativeView>, IMoveController {

void IMoveController .SetMovedDelegate(Action moved) { ... }
void IMoveController .RemoveMovedDelegate() { ... }

}
```

1. ViewHandler는 `SetMovedDelegate`를 여러번 하더라도 마지막 등록한 action만 수행하도록함
2. ViewHandler는 `RemoveMovedDelegate`시 등록된 action이 없더라도 에러 없이 처리함

이렇게 하면 

```suggestion
			if (changed)
			{
				nativeView.EvasMap = map;
				if (view.Handler is IMoveController controller) {
				    controller.SetMovedDelegate(OnMoved);
				}
			}
			else {
				if (view.Handler is IMoveController controller) {
				    controller.RemoveMovedDelegate();
				}
			}
```

Reply:
좀 더 확인해보니, 
ViewHandler.Tizen.cs 안에서 `UpdateTransformation`를 호출하고 있기 때문에, `SetMovedDelegate`/`RemoveMovedDelegate`를 ViewHandler에 추가하고 여기서 아래와 같이 처리하는게 좋을듯하네요, 

```  c#
public static void UpdateTransformation(this EvasObject nativeView, IView? view, out bool changed)


		internal static void UpdateTransformation(ViewHandler handler, IView view)
		{
			((NativeView?)handler.WrappedNativeView)?.UpdateTransformation(view, out bool changed);
                        if (changed) {
                               handler.SetMovedDelegate(()=> UpdateTransformation(handler, view));
                        } else {
                               handler.RemoveMovedDelegate();
                        }
		}
```

Reply:
WrappedNative와 NativeView가 서로 변경될수도 있는 상황에서, 그냥 static Dictionary로 evas object를 key로 사용하여 view별로 moved 핸들러를 관리할 수 있도록 수정함

Reply:
이 구조와는 상관없이 rotation시 view가 사라지는 이슈가 있음